### PR TITLE
Adds Economical Burrito

### PR DESCRIPTION
**Pull request by BurgerLUA**
# Overview
Adds a more economical burrito that can be produced with 1 meatball, 1 tortilla, and 5 water in the microwave.
Burrito

### CODE REVIEW

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -4656,6 +4656,14 @@
 	. = ..()
 	reagents.add_reagent("protein", 6)
 
+/obj/item/weapon/reagent_containers/food/snacks/burrito_pathetic
+	name = "pathetic burrito"
+	desc = "A lousy amount of meat wrapped in too much tortilla. The flavor balance is way off because of the lack of meat, not to mention it looks incredibly unappealing. Honestly you should straight up space the chef if they present you with this item, not even joking; if a chef lays this out on the counter, they're fucking valid. After you stuff them in the gibber, use their pathetic remains to make yourself a god damn proper meat burrito."
```
Comment (NortonDK) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: A valid? come on

Comment (BurgerLUA) [reply code/modules/reagents/reagent_containers/food/snacks.dm]: what

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 2]
```diff
@@ -4658,13 +4658,25 @@
...
+		"Honestly, what the fuck is this? Seriously? What the fuck is this? It's a fucking joke, that's what. The time and effort spent on this tiny monster is a fucking joke. Honestly you'd be better off eating crumbs off the kitchen floor than to seek any sustenance from this. I'm pretty fucking sure that tofu has more nutritional value than this micropenis of a burrito.",
+		"Okay, stop. You looked at it now, now it's your responsiblity to take action and deal with this joke. Throw it in the trash, <b>right now</b>. This isn't a joke, this is your brain telling you right now to dispose of this mess immediately and find yourself some actual fucking proper food.",
+		"Wait, does that have any meat? Wait, what the fuck? Where is the fucking meat? Where is the filling? At least vegans fucking attempt to put filling in burritos, such as carrots, but this fucking thing barely has any filling at all. You're pretty fucking sure that NanoTrasen paid good money for the chef, yet they're cutting corners like this. God fucking damn."
+	)
```
[the remark this answers is not in the document]
Comment (BurgerLUA) [reply code/modules/reagents/reagent_containers/food/snacks.dm]: descriptions are supposed to be descriptive

[the remark this answers is not in the document]
Comment (Pacmandevil) [reply code/modules/reagents/reagent_containers/food/snacks.dm]: Obviously the description should just follow the lines of the other ones. "it's a [name]" should be adequate.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 3]
```diff
@@ -4656,6 +4656,14 @@
 	. = ..()
 	reagents.add_reagent("protein", 6)
 
+/obj/item/weapon/reagent_containers/food/snacks/burrito_pathetic
+	name = "skinny burrito"
+	desc = "A more economical burrito for the masses."
+	icon_state = "pathetic"
+	bitesize = 6 //Easy to bite
```
[the remark this answers is not in the document]
Comment (Pacmandevil) [reply code/modules/reagents/reagent_containers/food/snacks.dm]: The whole point of it is a pathetic burrito. a burrito with one meatball could probably be eaten with one bite, I'd have to attempt it. I'll check back once I do though

[the remark this answers is not in the document]
Comment (Arrow768) [reply code/modules/reagents/reagent_containers/food/snacks.dm]: Hm, the normal burrito has a bitesize of 4 -> Requiring two bites to eat it.
So I see two options:
- Reduce the bitesize of the normal burrito, so you cant eat it with two bites
- Leave the bitesize for the mini-burrito as is. (6)

**Comment by JackBewt** [comment]
Now that it looks like a genuine pr without textwalls for descriptions, this is fine.

**Comment by skull132** [comment]
By decree of Jackboot, the goals of the thread have been met by Alb's PR. Closing this.

**Pull request closed without merging**